### DISCUSSION TITLE
Controlling Actors remotely using APIs

### DISCUSSION THREAD

**Discussion by KingAlawaka** [carla-simulator/carla#5830] [Q&A]
Hi all,

I am trying to build a simulation with an autonomous vehicle that gets messages from an outside web interface. 
As an example, the vehicle will drive automatically. If an external API call sends a message like, give way to all emergency vehicles based on that vehicle's behaviour will change. It is the same as sending a message from a vehicle to traffic lights, saying this is an emergency, and making it green for that vehicle's requested direction.
I want to know if there are any examples of something like that. 

Thank you.
Pasindu

**Comment by PooriyaTT** [comment]
Hey @KingAlawaka, have you found any solution for that? Because I am also trying to send the messages from RSU to the ego vehicle to act accordingly.
